### PR TITLE
Add rotation tf for bumblebee

### PR DESCRIPTION
**Pull request by staylo32**
Added rotation tf for bumble stereo camera and changed camera link name in tf from stereo_camera to bumblebee_mount_link

### CODE REVIEW

**Comment by bsubei** [comment]
Yay you finally figured out the stupid Quaternion thing? What was the problem?

**Comment by bsubei** [comment]
And why did you change the camera frame_id? because of the Gazebo plugin?

File: robot_setup_tf/src/broadcaster/tf_broadcaster.cpp [hunk 1]
```diff
@@ -35,8 +36,8 @@ int main(int argc, char** argv)
   		        ros::Time::now(),"base_link", "laser"));
 
 			broadcaster.sendTransform(
-				tf::StampedTransform(tf::Transform(tf::Quaternion(0, 0, 0, 1), tf::Vector3(0.1905, 0.0, 0.6112)),
-  		        ros::Time::now(),"base_link", "stereo_camera"));
+				tf::StampedTransform(tf::Transform(tf::Quaternion(-M_PI_2,0,M_PI_2), tf::Vector3(0.1905, 0.0, 0.6112)),
```
Comment (bsubei) [inline robot_setup_tf/src/broadcaster/tf_broadcaster.cpp:39]: Oh wow! It seems you just used the Quaternion constructor that uses Euler angles! Nice!

**Comment by staylo32** [comment]
Yes, the name was changed because of the plugin.

**Pull request merged**